editor/editor_spin_slider.cpp

```diff
@@ -255,7 +255,15 @@ void EditorSpinSlider::_notification(int p_what) {
 
 		draw_string(font, Vector2(Math::round(sb->get_offset().x), vofs), label, HALIGN_LEFT, -1, font_size, lc * Color(1, 1, 1, 0.5));
 
-		draw_string(font, Vector2(Math::round(sb->get_offset().x + string_width + sep), vofs), numstr, HALIGN_LEFT, number_width, font_size, fc);
+		Vector2 text_ofs = Vector2(Math::round(sb->get_offset().x + string_width + sep), vofs);
+		draw_string(font, text_ofs, numstr, HALIGN_LEFT, number_width, font_size, fc);
+
+		if (suffix != String()) {
+			int sw = font->get_string_size(numstr).width;
+			text_ofs.x += sw;
+			fc.a *= 0.4;
+			draw_string(font, text_ofs, suffix, HALIGN_LEFT, MAX(0, number_width - sw), font_size, fc);
+		}
```

Comment:
Seems like `EditorSpinSlider` margins are hard-coded for left-to-right layout, and it's not properly aligned when displayed in right-to-left layout (I'll fix it a bit later).

But I'm not about suffix placement, probably it should be on the right in both LTR and RTL layouts if units are not translated (and for unit symbols like degree), if it's translated units it should be flipped.

If it stays limited to the `px` and `°`, it's OK. If not, it probably better to handle it as a singe `00 unit` string and let ICU decide which order is right (with coloring in the custom `draw_glyph` loop).

Reply:
It will most likely remain limited to numbers (For now at least), but feel free to fix it later on if you believe it's worth it.

Reply:
I wanted at some point for some properties have the option to add strings instead of specific values, like -1 -> Disabled but will do it in a subsequent PR at some point in the future, which may not need RTL either.

Reply:
RTL layout follow-up PR - #50030